indico/modules/events/registration/controllers/management/sections.py

```diff
@@ -93,8 +106,12 @@ class RHRegistrationFormToggleSection(RHManageRegFormSectionBase):
 
     def _process_POST(self):
         enabled = request.args.get('enable') == 'true'
-        if not enabled and self.section.type == RegistrationFormItemType.section_pd:
-            raise BadRequest
+        if not enabled:
+            if self.section.type == RegistrationFormItemType.section_pd:
+                raise BadRequest
+            for field in self.section.children:
+                if field.is_condition:
+                    raise NoReportError.wrap_exc(BadRequest(_('Fields used as conditional cannot be disabled')))
```

Comment:
```suggestion
            if any(f.is_condition for f in self.section.children):
                raise NoReportError.wrap_exc(BadRequest(_('Sections with fields used as conditional cannot be disabled')))
```

Reply:
Done in [2c9bf92](https://github.com/indico/indico/pull/6678/commits/2c9bf92b6dc9c0dc1146317cc0f1cc2a636b5fa5).